### DISCUSSION TITLE
part-ii/2020-1/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#278] [Question comments]
Part II 2020 Algebraic Geometry: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2020-1/

**Comment by artie2000** [comment]
All nonsingular quadric surfaces are isomorphic. This is because they all have full rank, and so are related by a change of coordinates.

(a): This was on ES2.

(b): This was also on ES2 (use the Segre surface).

(c): I used the $P+tQ$ definition of a line and argued by contradiction.
